nix/package.nix

```diff
@@ -68,9 +68,9 @@ buildDotnetModule rec {
     inherit version;
   };
 
-  # SDK 6.0 required for Robust.LoaderApi
-  dotnet-sdk = with dotnetCorePackages; combinePackages [ sdk_8_0 sdk_6_0 ];
-  dotnet-runtime = dotnetCorePackages.runtime_8_0;
+  # SDK 8.0 & 6.0 required for Robust.LoaderApi
+  dotnet-sdk = with dotnetCorePackages; combinePackages [ sdk_9_0 sdk_8_0 sdk_6_0 ];
```

Comment:
I don't think `Robust.LoaderApi` needs the .NET 6 anymore, so that can be removed.

Reply:
Fixed. I was testing SDK versions but guess I didn't remove .NET 6 in the permutations.